modules/core/02-client/types/genesis_test.go

```diff
@@ -141,33 +134,13 @@ func (suite *TypesTestSuite) TestValidateGenesis() {
 			),
 			expPass: false,
 		},
-		{
-			name: "invalid client",
```

Comment:
why was this test case removed?

Reply:
I assumed the light client was invalid because it was a localhost client in this case. 

I can update the test case name to be more descriptive (to not confuse others) and put it back in.

Reply:
Hmm, that might be true, but maybe we can use it for `!ps.Params.AllowedClient` check

Reply:
Updated :handshake: